### PR TITLE
DISPATCH-1067 - updates to policy doc (updated)

### PR DESCRIPTION
**Pull request by bhardesty**

### CODE REVIEW

File: docs/books/user-guide/configuration-security.adoc [hunk 1]
```diff
@@ -734,57 +811,57 @@ vhost {
 
 <3> Any user can connect to this vhost. Users that are not part of the `admin` group are assigned to the `$default` group.
 
-<4> If the `admin-01` or `admin-02` user connects to the vhost, they are assigned to the `admin` user group.
+<4> If the `admin1` or `admin2` user connects to the vhost, they are assigned to the `admin` user group.
 
 <5> Users in the `admin` user group must connect from localhost. If the admin user attempts to connect from any other host, the connection will be denied.
 
-<6> Users in the admin user group can send and receive from any address offered by the vhost.
+<6> Users in the admin user group can send to any address offered by the vhost.
```
Comment (fgiorgetti) [inline docs/books/user-guide/configuration-security.adoc]: s/send to/receive from

File: docs/books/user-guide/configuration-security.adoc [hunk 2]
```diff
@@ -734,57 +811,57 @@ vhost {
...
 
-<7> Any non-admin user is permitted to connect from any host.
+<7> Users in the admin user group can receive from any address offered by the vhost.
```
Comment (fgiorgetti) [inline docs/books/user-guide/configuration-security.adoc]: s/receive from/send to

File: docs/books/user-guide/understand-router-configuration.adoc [hunk 3]
```diff
@@ -51,6 +51,133 @@ sectionName {
...
 
+[id='methods-for-using-pattern-matching']
+== Methods for Using Pattern Matching and Wildcards
+
+The router configuration file supports pattern matching and wildcards to enable you to match multiple values for certain attributes. However, the syntax varies based on the type of entity that you are configuring. 
+
+[id='router-address-pattern-matching']
+=== Pattern Matching for Addresses
+
+In some router configuration scenarios, you may need to use pattern matching to match a range of addresses rather than a single, literal address. Address patterns match any address that corresponds to the pattern.
```
Comment (jenmalloy) [inline docs/books/user-guide/understand-router-configuration.adoc]: Change "may" to "might"

File: docs/books/user-guide/understand-router-configuration.adoc [hunk 4]
```diff
@@ -51,6 +51,133 @@ sectionName {
...
+
+An address pattern is a sequence of tokens (typically words) that are delimited by either `.` or `/` characters. They also may contain special wildcard characters that represent words:
```
Comment (jenmalloy) [inline docs/books/user-guide/understand-router-configuration.adoc]: Change "may" to "might"

File: docs/books/user-guide/configuration-security.adoc [hunk 5]
```diff
@@ -442,290 +442,367 @@ For more information about these attributes, see xref:adding-sasl-authentication
...
 As an alternative to using the router configuration file, you can configure vhost policies in JSON files. If you have multiple routers that need to share the same vhost configuration, you can put the vhost configuration JSON files in a location accessible to each router, and then configure the routers to apply the vhost policies defined in these JSON files.
 
+.Prerequisites
+
+* Vhost policies must be enabled for the router. For more information, see xref:enabling-vhost-policies[].
+
 .Procedure
 
-. Determine where to store the vhost policy JSON files.
+. In the router configuration file, specify the directory where the vhost policy definition JSON files should be located.
```
[the remark this answers is not in the document]
Comment (bhardesty) [reply docs/books/user-guide/configuration-security.adoc]: It's actually where you want them to be stored. I changed this to make it more clear.

File: docs/books/user-guide/understand-router-configuration.adoc (hunk 3 above)
Comment (bhardesty) [reply docs/books/user-guide/understand-router-configuration.adoc]: Done.

File: docs/books/user-guide/understand-router-configuration.adoc (hunk 4 above)
Comment (bhardesty) [reply docs/books/user-guide/understand-router-configuration.adoc]: Changed to "can"

**Comment by bhardesty** [comment]
@ganeshmurthy this is ready to merge.

**Pull request closed without merging**